wordpress-mobile/WordPress-iOS
# Add feature flag + Create Button

#13320

Adds the new Floating Action Button to create a post. This is mostly to check the visual design of the button with other functionality coming in follow up PRs.

#### iPhone
![iPhone](https://user-images.githubusercontent.com/3250/74556553-8908fb80-4f1b-11ea-8df8-61e54bfa3f8c.png)

#### iPad
![iPad](https://user-images.githubusercontent.com/3250/74558193-126dfd00-4f1f-11ea-9037-8ebc85303f68.png)

### Notes
- Button is shown on all screens. To simplify review, a separate PR will properly hide the button depending on which screen is shown.
- There is no list sliding up from the bottom to select a Site Page or Blog Post. This will be added in a separate PR.

### Testing

- Open Application to any tab
- Notice Create Button in bottom right

PR submission checklist:
